scheduler/src/cook/mesos.clj

```diff
@@ -35,7 +35,8 @@
             [metatransaction.utils :as dutils]
             [metrics.counters :as counters]
             [swiss.arrows :refer :all])
-  (:import [org.apache.curator.framework.recipes.leader LeaderSelector LeaderSelectorListener]
+  (:import java.net.InetAddress
```

Comment:
Do we still need the `InetAddress` import?

Reply:
Done